src/content/docs/apm/agents/java-agent/installation/java-install-overview.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: An overview of the Java APM installation process
+tags:
+  - Agents
+  - Java agent
+  - Installation
+translate:
+  - jp
+metaDescription: An overview of how to download and install New Relic's APM agent for Java.
+---
+
+Our Java APM agent auto-instruments your code so you can start monitoring applications. Read this for an overview of the generic APM installation process. For more specific instructions, use [monitor your Java app](/docs/apm/agents/java-agent/getting-started/monitor-your-java-app) to find the specific resource for your Java app configuration.
+
+## 1. Prepare to install [#overview]
+
+Check the following:
+
+* Review the [compatibility and requirements](/docs/agents/java-agent/getting-started/compatibility-requirements-java-agent).
+* Check if your environment requires [additional or alternative install steps](/docs/agents/java-agent/additional-installation).
+
+When you're ready, use the guided installation for an automated install. If you're in the EU, select that option. Or for a manual approach, follow the instructions in this document to complete a basic Java agent installation. Either way, you need a New Relic account. ([It's free, forever](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#how-pricing-works).)
+
+<ButtonGroup>
+  <ButtonLink
+    data-tessen="stitchedPathLinkClick"
+    role="button"
+    to="https://newrelic.com/signup"
+    variant="primary"
+  >
+    Get an account
+  </ButtonLink>
+
+  <ButtonLink
+    data-tessen="stitchedPathLinkClick"
+    role="button"
+    to="https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1qYXZhLWludGVncmF0aW9uIiwiYWNjb3VudElkIjoyNjQwNDA5fQ==&platform[accountId]=1"
+    variant="primary"
+  >
+    Guided agent install
+  </ButtonLink>
+
+  <ButtonLink
+    role="button"
+    to="https://one.eu.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1qYXZhLWludGVncmF0aW9uIiwiYWNjb3VudElkIjoyNjQwNDA5fQ==&platform[accountId]=1"
+    variant="primary"
+  >
+    EU guided install
+  </ButtonLink>
+</ButtonGroup>
+
+## 2. Get the agent [#download-agent]
+
+Download `newrelic-java.zip` using `curl`, `Invoke-WebRequest` (PowerShell), or the New Relic UI:
+
+<CollapserGroup>
+  <Collapser
+    id="download-from-curl"
+    title={<>Download using <InlineCode>curl</InlineCode></>}
+  >
+    Complete the following:
+
+    1. Start a command-line session.
+    2. Change to a temporary directory where you can download the zip file.
+    3. Execute this `curl` command:
+
+       ```
+       curl -O https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip
+       ```
+    4. Unzip `newrelic-java.zip`
```

Comment:
Feel free to skip, but adding the unzip command here might simplify our users lives a bit (I personally always forget it and appreciate when it's in instructions)